os/include/tinyara/pm/pm.h

```diff
@@ -550,6 +569,7 @@ int pm_metrics(int milliseconds);
  * avoid so much conditional compilation in driver code when PM is disabled:
  */
 
+#define pm_start()              (0)
```

Comment:
should be #define pm_start() only , its a void funtion

Reply:
Done